src/screens/token/components/delegate-token/components/tabs/account-tab/account-tab.tsx

```diff
@@ -78,6 +85,18 @@ export class AccountTabComponent extends React.Component<
         this.props.fetchValidators(this.props.account, PosBasicActionType.DELEGATE);
         this.props.fetchDelegatedValidators(this.props.account);
         this.props.fetchAccountDelegateStats(this.props.account, this.props.token);
+        this.onFocus();
+    }
+
+    public onFocus() {
```

Comment:
Can change this to `getBalance()` and cover one more case: when navigate back from `Quick stake` should also call this.
Search in the project about focus. You'll find an example